Formula/x11vnc.rb

```diff
@@ -50,7 +50,7 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
- 
+
```

Comment:
@UniqMartin That's the thing, I never touched that line when I edited the test block but it's registering as a change... have you seen something like this before?